chromium/rules.js

```diff
@@ -328,9 +333,9 @@ RuleSets.prototype = {
 
     var tmp;
```

Comment:
BTW, this variable is not being used. Should I remove it in this PR?

Reply:
@cschanaj I will remove unused variables in one of my PRs.